connect-inject/handler_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
```suggestion
	"github.com/deckarep/golang-set"
```
I'm struggling to find how to fix this, but its a setting in the IDE to disable unnecessary import aliasing

Reply:
Ooh I'll look for the setting, thanks!

Reply:
These are the settings I have for imports:

<img width="993" alt="Screen Shot 2021-04-09 at 12 35 35 PM" src="https://user-images.githubusercontent.com/10640505/114225976-d5857680-9927-11eb-981c-120b17773885.png">

Reply:
I'm using the same settings and it won't remove that right now :(

Reply:
Doh! No worries :D